src/core_modules/capture-core/components/Pages/EnrollmentEditEvent/EnrollmentEditEventPage.epics.js

```diff
@@ -26,3 +28,24 @@ export const updateEventFailedEpic = (action$: InputObservable) =>
             return batchActions([saveFailed(), rollbackEnrollmentEvent(eventId)]);
         }),
     );
+export const saveEventAndCompleteEnrollmentSucceededEpic = (action$: InputObservable) =>
```

Comment:
Why were these 2 epics moved from `EnrollmentAddEventPage.epics` into `EnrollmentEditEventPage.epics.js`? 🤔 
I wasn't able to trigger them, what are the steps to do so? Also the `eventId` is known when editing so there is no need to send it in `commitEnrollmentAndEvents(meta.uid, eventId)`

Reply:
No, see my original comment further up. Do you think it safe to remove?

Reply:
Yes, I bealive is safe to remove these 2 epics (they are not being triggered at the moment). When editing the event the commit and rollback are currently being handled in `editEventDataEntry.epics` by [saveEditedEventSucceededEpic](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WidgetEventEdit/EditEventDataEntry/editEventDataEntry.epics.js#L171) and [saveEditedEventFailedEpic](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WidgetEventEdit/EditEventDataEntry/editEventDataEntry.epics.js#L202)